manifests/init.pp

```diff
@@ -40,6 +40,8 @@
 #   The directory used to install the report processor to
 # @param use_client_tls_certs
 #   Enable client TLS authentication to foreman
+# @param watch_for_facts_and_publish_them
```

Comment:
I really had no idea for a good name. If someone has a suggestion, please let me know.

Reply:
`fact_watcher_service` is the only thing I can think of now

Reply:
updated it